src/lib/offline-api/index.ts

```diff
@@ -332,17 +340,17 @@ class OfflineAPI {
     // Store clone as a request
     this.currentRequestsRecorded.push(deleteRequest(ct.clone()))
 
-    await this.modifiedContentTypes.delete(id)
-    await this.publishedContentTypes.delete(id)
-    await this.savedContentTypes.delete(id)
+    this.modifiedContentTypes.delete(id)
+    this.publishedContentTypes.delete(id)
+    this.savedContentTypes.delete(id)
 
     // TODO: Is the DeleteContentType hook actually being implemented?
-    for (const validator of this.contentTypeValidators) {
-      if (validator.hooks.includes(ApiHook.DeleteContentType)) {
-        const errors = validator.validate(ct, this.savedContentTypes.get(id), this.publishedContentTypes.get(id))
-        this.currentValidationErrorsRecorded = this.currentValidationErrorsRecorded.concat(errors)
-      }
-    }
+    // for (const validator of this.contentTypeValidators) {
```

Comment:
So should we remove this or not?

Reply:
Ah, this was added again by mistake, I think it should be removed, because it is not implemented. I will do that.